src/collectors/uniswapx_order_collector.rs

```diff
@@ -171,11 +187,113 @@ mod tests {
             base_url: url.clone(),
             chain_id: 1,
             order_type: order_type,
+            // Inconsequential query parameter because we mock the order service response
+            execute_address: "0x0000000000000000000000000000000000000000".to_string(),
         };
 
         (res, server, mock)
     }
 
+    // Helper mock order objects
+    fn create_mock_priority_order() -> PriorityOrder {
```

Comment:
nit: think you can be everything from here under `#[cfg(test)]` so that they don't get compiled unless you run `cargo test`